### PR TITLE
Fix the logging issues

### LINKED ISSUE #1: 'No handlers could be found for logger "__main__" ' is displayed when an error happened
It seems like the logger is not correctly configured.

### PR DESCRIPTION
**Pull request by askeing**
- fix #1

### CODE REVIEW

**Comment by askeing** [comment]
@JohanLorenzo @shakoho r?

File: taskcluster_util/model/credentials.py [hunk 1]
```diff
@@ -7,11 +9,12 @@ def __init__(self, **kwargs):
         self.clientId = kwargs['clientId']
         self.accessToken = kwargs['accessToken']
 
+        self.logger = logging.getLogger(__name__)
         try:
             self.certificate = kwargs['certificate']
         except KeyError:
             # TODO: Replace by log.warning once the loggers works.
```
Comment (JohanLorenzo) [inline taskcluster_util/model/credentials.py]: Nit: We can remove this comment :)

**Comment by JohanLorenzo** [comment]
For consistency, I think it's better if we migrate every `print` to `logger.info`. Otherwise we end up with lines like:

```
2015-07-28 12:09:17,022 - model.credentials - WARNING - No certificate is present in the given credentials. If the credentials are temporary you won't be able to download anything
### Finding the TaskID of Namespace [gecko.v1.mozilla-central.latest.linux.aries-eng.opt] ...
2015-07-28 12:09:17,030 - requests.packages.urllib3.connectionpool - INFO - Starting new HTTPS connection (1): index.taskcluster.net
### The TaskID of Namespace [gecko.v1.mozilla-central.latest.linux.aries-eng.opt] is [gHLE0bl7SlS9HWfCKvs_1Q].
```

Also, I think the timestamps, the log level and the package are a bit too much for the end user. I think we can set the format to `'%(message)s'` in regular mode, and to `'%(asctime)s - %(name)s - %(levelname)s - %(message)s'` when we're in debug mode. @askeing What do you think?

File: taskcluster_util/model/credentials.py [hunk 2]
```diff
@@ -7,11 +9,12 @@ def __init__(self, **kwargs):
         self.clientId = kwargs['clientId']
         self.accessToken = kwargs['accessToken']
 
+        self.logger = logging.getLogger(__name__)
```
Comment (JohanLorenzo) [inline taskcluster_util/model/credentials.py]: I think we can set the logger at the class level. Example:

``` python
class Credentials(dict):

    logger = logging.getLogger(__name__)

    def __init__(self, **kwargs):
```

File: taskcluster_util/taskcluster_download.py [hunk 3]
```diff
@@ -41,9 +42,12 @@ def __init__(self):
         artifact_group = parser.add_argument_group('Download Artifact', 'The artifact name and dest folder')
         artifact_group.add_argument('-a', '--artifact', action='store', dest='aritfact_name', help='The artifact name on Taskcluster')
         artifact_group.add_argument('-d', '--dest-dir', action='store', dest='dest_dir', help='The dest folder')
+        parser.add_argument('--debug', action='store_true', dest='debug', default=False, help='Enable debug mode.')
```
Comment (JohanLorenzo) [inline taskcluster_util/taskcluster_download.py]: How about defining `-v` `--verbose`? It would look more standard to me

**Comment by askeing** [comment]
@JohanLorenzo @shakoho r?
- modify the argument name from `debug` to `verbose`
- modify the logger of credentials from object level to class level
- setup different logger config base on `verbose` argument

**Comment by ShakoHo** [comment]
r+, except using logger instead of print...after discussion with Askeing, that will be on another PR

**Comment by JohanLorenzo** [comment]
r+ to me too. I agree on the logger vs print point, it'll be clearer in another PR.

**Pull request merged**